imports/plugins/included/ui-search/lib/components/searchModal.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
```

Comment:
If you haven't made any change to this file, can you revert back to what is on staging branch? That is copy and paste the content here.  There are quality issues in this file, which were fixed in one of the PRs merged to staging. Example is the difference in [this lines](https://github.com/andela/annie-hall-rc/blob/staging/imports/plugins/included/ui-search/lib/components/searchModal.js#L11-L47)

Reply:
Fixed!